src/insns/csrr_32bit.adoc

```diff
@@ -63,18 +63,18 @@ xref:extended_CSR_writing[xrefstyle=short] for the action taken on writing an XL
 +
 If `cd` is `c0` (or `rd` is `x0`), then <<CSRRWI>> shall not read the CSR and
 and shall not cause any of the side effects that might occur on a CSR read. If
-`rs1` is `x0`, then <<CSRRS>>, <<CSRRC>>, <<CSRRSI>> and <<CSRRCI>> will not
-write to the CSR at all, and so shall not cause any of the side effects that
-might otherwise occur on a CSR write.
+`rs1` is `x0` for <<CSRRS>> and <<CSRRC>>, or `imm` is 0 for <<CSRRSI>> and
```

Comment:
```suggestion
`rs1` is `x0` for <<CSRRS>> and <<CSRRC>>, or `uimm` is 0 for <<CSRRSI>> and
```

Just looked at the unpriv spec and it uses `uimm`. Does not need to be fixed here, but should probably clean this up later.